packages/frontend/core/src/modules/collection/entities/collection.ts

```
@@ -0,0 +1,87 @@
+
+export class Collection extends Entity<{ id: string }> {
+  constructor(
+    private readonly store: CollectionStore,
+    private readonly rulesService: CollectionRulesService
+  ) {
+    super();
+  }
+
+  id = this.props.id;
+
+  info$ = LiveData.from<CollectionInfo>(
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)